### DISCUSSION TITLE
Installing Poetry in a Docker container no longer working

### DISCUSSION THREAD

**Discussion by radumarg** [python-poetry/poetry#6755] [Q&A] [answered]
I am using Poetry to install a python project using Poetry in a Docker container. Everything used worked fine until I switched to the latest version of Poetry (1.2.1). Now when I run my container, after starting a python shell, when I want to import any module installed by the project I get: **ModuleNotFoundError**: No module named xxxx. 

The poetry install works fine, I attached to a running container and run it myself and found that it works without problems.  I tried installing my project globally in the system environment and using a virtual env. In both cases, I get the same error: module not found when trying to import a module. I double-checked and verified that I am using the correct python executable when opening a python shell. 

The docker file I used is listed here: [https://stackoverflow.com/questions/73997844/installing-a-python-project-using-poetry-in-a-docker-container](https://stackoverflow.com/questions/73997844/installing-a-python-project-using-poetry-in-a-docker-container)

What can I do to debug this issue?

**Comment by neersighted** [comment, marked as answer]
Looks like https://github.com/python-poetry/poetry/issues/6459

**Reply by radumarg** [reply]
Thanks!
